File: Linguagem C/1015.c
#include <stdio.h>
#
int main()
{
    float x1,y1,x2,y2,d;

    scanf("%f %f %f %f",&x1,&y1,&x2,&y2);

    d = ((x2-x1)*(x2-x1))+((y2-y1)*(y2-y1));

    printf("%.4f\n",sqrt(d));
    return 0;
}

File: Linguagem GO/1015.go
package main

import (
	"fmt"
	"math"
)

func main() {
	
	var X1, X2, Y1, Y2, distancia float64

	fmt.Scanf("%f",&X1)
	fmt.Scanf("%f",&Y1)
	fmt.Scanf("%f",&X2)
	fmt.Scanf("%f",&Y2)
	
	distancia = math.Sqrt(math.Pow((X2 - X1), 2) + math.Pow((Y2 - Y1), 2))

	fmt.Printf("%.4f\n",distancia)
}
